woocommerce/woocommerce-android
# Issue/3662 add customer list

Resolves #3662

Add customer screen with a list of customers

**Testing**
* Go to the orders screen
* Click FAB
* Click "START CUSTOMER CREATION"
* Add customer screen should be visible with customers

Video:

https://user-images.githubusercontent.com/4923871/110446956-cb761b00-80d0-11eb-8ba0-9b93a54a9907.mp4


Update release notes:
